app/gradlew.bat

```diff
@@ -0,0 +1,91 @@
+@rem
```

Comment:
DITTO

Reply:
Shall I edit this file under project directory (~\android\app)? It seems unavailable form IDE level.